packages/messaging/README.md

```diff
@@ -109,259 +111,298 @@ public class MainApplication extends Application implements ReactApplication {
+                Identity.EXTENSION,
+                Optimize.EXTENSION,
+                Messaging.EXTENSION);
+    MobileCore.registerExtensions(extensions, o -> {
+      MobileCore.lifecycleStart(null);
+    });
   }
-}     
+}
 ```
 
 ### Importing the extension:
 
 ```javascript
-import { Messaging, MessagingDelegate, MessagingEdgeEventType, Message } from '@adobe/react-native-aepmessaging';
+import {
+  Messaging,
+  MessagingDelegate,
+  MessagingEdgeEventType,
+  Message,
+} from "@adobe/react-native-aepmessaging";
 ```
 
 ## API reference
+
 ### extensionVersion
+
 Returns the version of the AEPMessaging extension
 
 **Syntax**
+
 ```javascript
 extensionVersion(): Promise<string>;
 ```
 
 **Example**
+
 ```javascript
-Messaging.extensionVersion().then(version => console.log("AdobeExperienceSDK: Messaging version: " + version));
+Messaging.extensionVersion().then((version) =>
+  console.log("AdobeExperienceSDK: Messaging version: " + version)
+);
 ```
 
 ## Configure Adobe Journey Optimizer
+
 To configure Adobe Journey optimizer Messaging in Launch follow steps in [Configure Adobe Journey optimizer](https://developer.adobe.com/client-sdks/documentation/adobe-journey-optimizer/#setup-adobe-journey-optimizer-extension)
 
 ## Push Notification Setup
+
 Handling push notifications must be done in native (Android/iOS) code for the React Native app. To configure push notification in the native project, follow the instructions provided by their respective platforms:
-- [Apple - iOS push notification setup](https://developer.apple.com/documentation/usernotifications/registering_your_app_with_apns)  
+
+- [Apple - iOS push notification setup](https://developer.apple.com/documentation/usernotifications/registering_your_app_with_apns)
 - [Google - Android push notification setup](https://firebase.google.com/docs/cloud-messaging/android/client)
 
 ## Push Messaging APIs usage
-Push messaging APIs in the SDK must be called from the native Android/iOS project of React Native app.  
 
-###### [iOS API usage](https://github.com/adobe/aepsdk-messaging-ios/blob/main/Documentation/APIUsage.md)  
+Push messaging APIs in the SDK must be called from the native Android/iOS project of React Native app.
+
+###### [iOS API usage](https://github.com/adobe/aepsdk-messaging-ios/blob/main/Documentation/APIUsage.md)
 
 ##### [Android API usage](https://github.com/adobe/aepsdk-messaging-android/blob/main/Documentation/APIUsage.md)
+
 In Android, [MessagingPushPayload](https://github.com/adobe/aepsdk-messaging-android/blob/main/Documentation/push/MessagingPushPayload.md#messagingpushpayload-usage) can be used for getting the notification attributes like title, body, and action. These are useful for push notification creation.
 
 ## In-app messages API reference
+
 ### refreshInAppMessages
+
 Initiates a network call to retrieve remote in-app message definitions.
 
 **Syntax**
+
 ```javascript
-refreshInAppMessages()
+refreshInAppMessages();
```

Comment:
For Syntax we don't need ; Same for others in this doc.

Reply:
`;` is optional in TypeScript. I think we can just keep it and let messaging team decide on this.